.github/workflows/build.yaml

```diff
@@ -0,0 +1,43 @@
+name: make docker-build
+
+on:
+  pull_request:
```

Comment:
this should be no on the pull. it should be on push, and change the branches to main for now.